lib/docker/artifact_image.js

```diff
@@ -15,6 +16,21 @@ import sleep from '../util/sleep';
 
 let debug = Debug('docker-worker:artifactImage');
 
+async function decompressLz4File(inputFile) {
+  let outputFile = path.join(path.dirname(inputFile), path.basename(inputFile, '.lz4'));
```

Comment:
basename strips the provided extension off if ti's the last extension of the filename.  So path.basename('image.tar.lz4', '.lz4') becomes 'image.tar'